tests/os_snapshot_test/os_snapshot_windows_test.go

```diff
@@ -54,11 +55,42 @@ func TestShadowCopy(t *testing.T) {
 
 	oid := sources[0].Snapshots[0].ObjectID
 	entries := clitestutil.ListDirectory(t, e, oid)
+	t.Log("sources[0].Snapshots[0].ObjectID entries:", entries)
 
 	if isAdmin {
+		require.NotEmpty(t, entries)
```

Comment:
Before, there was a panic in cases where `entries` was empty. This explicitly checks the condition and stops the test if the condition is not met making it easier to diagnose the problem.